File: Lib/Telerik/Examples/QuickStart/Map/FirstLook/TripCity.vb
Imports Telerik.WinControls.UI.Map

Namespace Telerik.Examples.WinControls.Map.FirstLook
	Public Class TripCity
'INSTANT VB NOTE: The variable location was renamed since Visual Basic does not allow class members with the same name:
		Private location_Renamed As PointG
'INSTANT VB NOTE: The variable name was renamed since Visual Basic does not allow class members with the same name:
		Private name_Renamed As String
'INSTANT VB NOTE: The variable tripStop was renamed since Visual Basic does not allow class members with the same name:
		Private tripStop_Renamed As Integer
'INSTANT VB NOTE: The variable isFlight was renamed since Visual Basic does not allow class members with the same name:
		Private isFlight_Renamed As Boolean

		Public Sub New(ByVal name As String, ByVal location As PointG, ByVal tripStop As Integer, ByVal isFlight As Boolean)
			Me.name_Renamed = name
			Me.location_Renamed = location
			Me.tripStop_Renamed = tripStop
			Me.isFlight_Renamed = isFlight
		End Sub

		Public Property Location() As PointG
			Get
				Return location_Renamed
			End Get
			Set(ByVal value As PointG)
				Me.location_Renamed = value
			End Set
		End Property

		Public Property Name() As String
			Get
				Return name_Renamed
			End Get
			Set(ByVal value As String)
				Me.name_Renamed = value
			End Set
		End Property

		Public Property TripStop() As Integer
			Get
				Return tripStop_Renamed
			End Get
			Set(ByVal value As Integer)
				Me.tripStop_Renamed = value
			End Set
		End Property

		Public Property IsFlight() As Boolean
			Get
				Return isFlight_Renamed
			End Get
			Set(ByVal value As Boolean)
				Me.isFlight_Renamed = value
			End Set
		End Property
	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/Map/FirstLook/TripCity.cs
using Telerik.WinControls.UI.Map;

namespace Telerik.Examples.WinControls.Map.FirstLook
{
    public class TripCity
    {
        private PointG location;
        private string name;
        private int tripStop;
        private bool isFlight;

        public TripCity(string name, PointG location, int tripStop, bool isFlight)
        {
            this.name = name;
            this.location = location;
            this.tripStop = tripStop;
            this.isFlight = isFlight;
        }

        public PointG Location
        {
            get { return location; }
            set { this.location = value; }
        }

        public string Name
        {
            get { return name; }
            set { this.name = value; }
        }

        public int TripStop
        {
            get { return tripStop; }
            set { this.tripStop = value; }
        }

        public bool IsFlight
        {
            get { return isFlight; }
            set { this.isFlight = value; }
        }
    }
}
